docs/use-cases/_index.md

```diff
@@ -20,4 +19,4 @@ Select the card for a step-by-step guide to the solution:
 {{% card link="/use-cases/manage-fleet/" class="fit-contain" %}}
 {{< /cards >}}
 
-Find more examples of how Viam is being used in the world by reviewing [Customer Stories](https://www.viam.com/customers), [tutorials](/tutorials/), or [blog posts](https://www.viam.com/blog?categories=Tutorials).
+If you're looking for examples of how Viam is being used in the world, have a look at [customer stories](https://www.viam.com/customers), [step-by-step tutorials](/tutorials/), or [blog posts](https://www.viam.com/blog?categories=Tutorials).
```

Comment:
Supernit: to force some non-linked text between links (to help convey three different types of content | three different links), how about:

```suggestion
If you're looking for examples of how Viam is being used in the world, have a look at [customer stories](https://www.viam.com/customers), follow our [step-by-step tutorials](/tutorials/), or browse our [blog posts](https://www.viam.com/blog?categories=Tutorials).
```